### DISCUSSION TITLE
RaspberryMatic with HmIP-HAP and no RPI-RF-MOD/HM-MOD-RPI-PCB

### DISCUSSION THREAD

**Discussion by sqoerk-dev** [OpenCCU/OpenCCU#1010] [🇺🇸 Q&A] [answered]
As the new eq3 CCU version supports the hmip Access Point as LAN device to connect the devices I would like to see RaspberryMatic Lite version without rf module support. Also it would be great starting it as a docker container on any device. I know this is no easy goal but it would be a great step.

Thanks in advance for all your work done. Even if this goal will never reached. Great job jens!!!!

**Comment by jens-maus** [comment, marked as answer]
Sorry, but this is not possible, hardware wise. For the HmIP-HAP (homematicIP Access Point) to work as a LAN Gateway with any kind of CCU (either CCU3, RaspberryMatic, piVCCU or even debmatic) a RPI-RF-MOD rf module is absolutely necessary. The reason being that the RPI-RF-MOD is not only used for RF communication, but the used coprocessor on this RF module is also used for the necessary AES encryption of the hmIP RF protocol. Thus, there is and will be no way of getting support for using a HmIP-HAP or even HmIPW-DRAP (for wired communication) without using the RPI-RF-MOD.

**Reply by sqoerk-dev** [reply]
Thank you very much for your answer. Good to know why it won't be possible without eq3 allowing it in any way.

I can't find a "close" button as it's no issue. If you have the possibility to lock this thread (all is said) please do so :)
